docs/source/modin/dataframe.rst

```diff
@@ -207,3 +207,10 @@ DataFrame
     DataFrame.to_pandas
     DataFrame.to_snowflake
     DataFrame.to_snowpark
+
+.. rubric:: Caching
```

Comment:
update the dataframe_supported and series_supported rst files as well

Reply:
Seems like we don't include to_snowpark/to_snowflake in those files - should we still include this even though it is a non-pandas API?